src/Hazelcast.Net.Examples/Program.cs

```diff
@@ -113,7 +113,7 @@ private static string ClientExceptionCode(Exception e)
         private static void InitializeUnobservedExceptions()
         {
             // make sure the queue is empty
-            while (_unobservedExceptions.TryDequeue(out _))
+            while (UnobservedExceptions.TryDequeue(out _))
```

Comment:
Can we use https://docs.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentqueue-1.clear?view=net-5.0#System_Collections_Concurrent_ConcurrentQueue_1_Clear here?

Reply:
yes, though that is only available with .NET Core and .NET, not .NET Framework.